configure.json

```diff
@@ -1,4 +1,29 @@
 {
+    "mss-check-hasredzone":{
+        "program": "mss-check-redzone",
+        "arguments": { "0": ["0"]},
```

Comment:
I would remove line 4 as well. For the basic test of redzone, run it without argument.